### DISCUSSION TITLE
Show more button on large filters list

### DISCUSSION THREAD

**Discussion by CushtyDev** [filamentphp/filament#13811] [Help]
### Package

Table builder

### Package Version

v3

### How can we help you?

Hi I am using filters on a table in a full page component, I have a large checkbox list with many options, is there a way to have a show more or make it collapable so not all options are shown right away like Airbnb does?

![Screenshot 2024-08-03 at 17 39 03](https://github.com/user-attachments/assets/a3d0c5d1-fc1f-4e24-a404-23a6bc28f6e9)

I am using a modal and it's getting very long, thanks

**Comment by micraux** [comment]
Hi,

You can create your own modal/view component:
[https://filamentphp.com/docs/3.x/support/blade-components/modal](https://filamentphp.com/docs/3.x/support/blade-components/modal)
